cypress/integration/features/components/editingCard.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Editing Card
```

Comment:
I wanted to try out the concept of Component Testing here, thus
`cypress/.../components/editingCard`